vendor/github.com/golang/protobuf/jsonpb/jsonpb.go

```diff
@@ -0,0 +1,829 @@
+// Go support for Protocol Buffers - Google's data interchange format
```

Comment:
This is a vendor update, does vendor.conf need update?

Reply:
Only update jsonpb here, I think we dont need update vendor.conf unless other model depend on the new version of protobuf.  :)